File: app/concepts/matestack/ui/core/form/radio/radio.rb
require_relative './base'

module Matestack::Ui::Core::Form::Radio
  class Radio < Base

    vue_js_component_name "matestack-ui-core-form-radio"

    def response
      div class: "matestack-ui-core-form-radio" do
        render_options
        render_errors
      end
    end

  end
end

File: app/concepts/matestack/ui/core/form/radio/radio.js
import Vue from "vue/dist/vue.esm";

import formRadioMixin from "./mixin";
import componentMixin from "../../component/component";

const componentDef = {
  mixins: [componentMixin, formRadioMixin],
  data() {
    return {};
  }
}

let component = Vue.component("matestack-ui-core-form-radio", componentDef);

export default componentDef;
